provider/aws/error_test.go

```diff
@@ -9,3 +16,10 @@ func (e errorNotFound) Error() string {
 func (e errorNotFound) NotFound() bool {
 	return true
 }
+
+func TestNoSuchBuild_Error(t *testing.T) {
+	err := aws.NoSuchBuild("B12345")
+	if assert.Error(t, err) {
+		assert.Equal(t, errors.New("no such build: B12345").Error(), err.Error())
```

Comment:
We could take advantage of `assert.EqualError` here and not wrap the string with a new error.

```
assert.EqualError(t, err,  expectedErrorString)
```